### PR TITLE
Pretty printer for audit

### PR DESCRIPTION
**Pull request by rbren**
Screen Shot 2021-03-08 at 9 17 03 AM

### CODE REVIEW

File: cmd/polaris/audit.go [hunk 1]
```diff
@@ -47,7 +48,8 @@ func init() {
 	auditCmd.PersistentFlags().IntVar(&minScore, "set-exit-code-below-score", 0, "Set an exit code of 4 when the score is below this threshold (1-100).")
 	auditCmd.PersistentFlags().StringVar(&auditOutputURL, "output-url", "", "Destination URL to send audit results.")
 	auditCmd.PersistentFlags().StringVar(&auditOutputFile, "output-file", "", "Destination file for audit results.")
-	auditCmd.PersistentFlags().StringVarP(&auditOutputFormat, "format", "f", "json", "Output format for results - json, yaml, or score.")
+	auditCmd.PersistentFlags().StringVarP(&auditOutputFormat, "format", "f", "json", "Output format for results - json, yaml, pretty, or score.")
+	auditCmd.PersistentFlags().BoolVar(&noColor, "no-color", false, "Whether to use color in pretty format.")
```
Comment (dominikmeyersap) [inline cmd/polaris/audit.go:52]: Just wondering, must it be negative? It basically means is a double negative. `no-color=false` = not having no color = color 😄

File: cmd/polaris/audit.go [hunk 2]
```diff
@@ -98,6 +100,8 @@ func runAndReportAudit(ctx context.Context, c conf.Configuration, auditPath, wor
 		if err == nil {
 			outputBytes, err = yaml.JSONToYAML(jsonBytes)
 		}
+	} else if outputFormat == "pretty" {
```
Comment (dominikmeyersap) [inline cmd/polaris/audit.go:103]: Nice, lead time of polaris is 6h! 👍

File: pkg/validator/output.go [hunk 3]
```diff
@@ -25,6 +28,17 @@ const (
 	PolarisOutputVersion = "1.0"
 )
 
+var (
+	successMessage = "🎉 Success"
+	dangerMessage  = "❌ Danger"
+	warningMessage = "😬 Warning"
```
Comment (dominikmeyersap) [inline pkg/validator/output.go:34]:
```suggestion
	warningMessage = "⚠️ Warning"
```
Maybe? 🤔

**Review by dominikmeyersap — COMMENTED** [review]
My two cents as the stakeholder of the hooman output 🐶 😆

File: pkg/validator/output.go (hunk 3 above)
Comment (rbren) [reply pkg/validator/output.go:34]: That's what I tried first!

Strangely the ⚠️ emoji gets mangled in my terminal. I wonder if I've just got a bad font, or if it's something that would affect other people...

File: cmd/polaris/audit.go (hunk 1 above)
Comment (rbren) [reply cmd/polaris/audit.go]: Great point!

**Comment by dominikmeyersap** [comment]
image
:D

**Comment by rbren** [comment]
😆 almost there! Just need a team review - hopefully today 🤞

**Review by makoscafee — APPROVED** [review]
LGTM. mind adding documentation of the flag may be in `infrastructure-as-code`?

**Review by baderbuddy — APPROVED** [review]
LGTM

**Comment by rbren** [comment]
Good call on the docs barnie!

**Pull request merged**